interview-questions.md

```diff
@@ -4,56 +4,60 @@ Answer the following questions. First, without external resources. Challenge you
 
 1. What is object destructuring?
 
-  Your answer:
+  Your answer: Object destructuring is a tool in JavaScript that lets you quickly and cleanly assign the properties of an object to variables you can work with in a function or block of code. 
 
-  Researched answer:
+  Researched answer: Destructuring can also be used with arrays and you can assign your own variable names to each value using a colon. 
 
 
 
 2. What is a DOM event?
 
-  Your answer:
+  Your answer: A DOM event is any interaction between the user and the interface. 
 
-  Researched answer:
+  Researched answer: They are events or interactions with the webpage that are detected by the DOM and used to trigger JavaScript. 
 
 
 
 3. What is the difference between a class and an object?
 
-  Your answer:
+  Your answer: A class is like a blueprint for making an object. They have predetermined properties and values that will be assigned to a new object that is created whenever that class' constructor is called. 
 
-  Researched answer:
+  Researched answer: An object is explicitly defined when initialized
 
 
 
 4. What is the difference between a HTML div and a span?
 
-  Your answer:
+  Your answer: I actually don't know. 
 
-  Researched answer:
+  Researched answer: A div is a general element used to structure HTML documents, while a span is an inline tag used to style a specific piece of text within an existing div or other element. 
 
 
 
 5. What is React.Fragment (or <>) and why would you want to use it?
 
-  Your answer:
+  Your answer: A react fragment is a an HTML container used in JSX for React that does not create nodes on the DOM when rendered. This allows you to wrap your entire HTML to be rendered in a lightweight, outer-most tag that React can enterpret. 
 
-  Researched answer:
+  Researched answer: React fragments can also be used to group components in the render to make your code cleaner. If you use the full declaration React.Fragment you may need to pass it a key. 
 
 
 
 6. What are three options for creating responsive design?
 
-  Your answer:
+  Your answer: Using the :hover method in CSS, using JavaScript to animate your sidebar (@keyframes), or by using mediaqueries to ensure the content matches the size of the viewing pane. 
```

Comment:
Your answer regarding the mediaqueries is totally on point, but the JS animations and :hover pseudo-selector wouldn't fit here.  I think there might be a bit of confusion on what Responsive means (Flexible viewing aspects), but it looks like you got it cleared up in your researched answer.